### DISCUSSION TITLE
Question relating to locations

### DISCUSSION THREAD

**Discussion by shulsmann** [snipe/snipe-it#11038] [Q&A]
We're exploring and trying out Snipe-IT to manage our inventory, but also equipment that we manage at various customers.

For example, we have on customer with around 60 locations where we manage firewalls and a primary and secondary ISP lines, we've added some locations in snipe-it, and we'd like to add the ISP provider and the customer id for that location.

So we're looking into the possibility of adding fields like this;

[ ISP ]  - [ ISP customer service ID ]

Does anyone know if this is possible and if so, point me into the right direction on how to accomplish this?

**Comment by uberbrady** [comment]
Hrm. Well, custom fields (and even encrypted custom fields) exist for assets. So maybe you could create a fieldset for "ISP Demarc" or "ISP Equipment" and associate that with the model of their cable modem or whatever thing they're using for internet, and then put the ISP and customer service ID in that? It's a few more clicks away than what you probably want, but it's the best way I can think of doing this without modifying the source (which we generally discourage, since it makes upgrades so utterly miserable).
